exercises/solutions/01-import-data.md

```diff
@@ -0,0 +1,16 @@
+## Import Data
+
+### Create Users
+```sql
+copy users(user_handle,first_name,last_name,email,gender,age,language) FROM '/Users/zac/Documents/01. Projects/advanced-sql-pt/user-data.csv' DELIMITER ',' CSV HEADER;
```

Comment:
We could make the paths generic. `/Users/{YOUR_USER}/{PATH_TO_PROJECT}/user-data.csv`

Reply:
good catch